homeassistant/components/netatmo/climate.py

```diff
@@ -213,13 +218,12 @@ def __init__(
 
         self._device_name = self._data.rooms[home_id][room_id]["name"]
         self._attr_name = f"{MANUFACTURER} {self._device_name}"
-        self._current_temperature: float | None = None
-        self._target_temperature: float | None = None
-        self._preset: str | None = None
+        self._attr_current_temperature: float | None = None
+        self._attr_target_temperature: float | None = None
+        self._attr_preset_mode: str | None = None
```

Comment:
We don't have to type it again :)

```suggestion
        self._attr_preset_mode = None
```

This could also be set as a class attribute